.gitignore

```diff
@@ -14,3 +14,5 @@ coverage
 
 # Package Lock - just not ready
 package-lock.json
+
+.tern-port
```

Comment:
global gitignore might be less work if you want to keep this out of all projects you work on. up to you
`git config --global core.excludesfile '~/.gitignore'`